spec/models/institution_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Institution, type: :model do
+  describe 'methods' do
+    it "has a full address method from it's components" do
+      type = create(:institution_type)
+      i = create(:institution, institution_type: type)
+      expect(i.address).to eq("#{i.city} #{i.street} #{i.building_no}/#{i.apartment_no} #{i.zip_code}")
```

Comment:
Here you're basically reusing and testing the implementation.
Tests should be dumb (vs. smart) and, above all, readable. It'd be better to explicitly pass values to the factory and then use an actual output string, something like:
```ruby
institution = create(:institution, city: 'Bielsko', street: 'Lompy', ...)
expect(institution.address).to eq("Bielsko Lompy ...")
```
That way, you're describing and testing the specification instead of the implementation, and it's immediately clear how does the address format look like